utils/update_checkout/__init__.py

```diff
@@ -1,4 +1,3 @@
-
-from update_checkout import main
+from update_checkout.update_checkout import main
```

Comment:
If you don't want the absolute import, it can also be done with `from .update_checkout import main`.

Reply:
This is another case of "the only solution I found that worked in both 2 and 3"

Reply:
```
from .update_checkout import main

__all__ = ["main"]
```

Just to let you know that worked when Im trying to compile the tensorflow branch... dont know if compatible with python2, changint the 2 `__init__.py` files for `from update_checkout.update_checkout import main` seem to not work on my end??